nixos/modules/virtualisation/amazon-image.nix

```diff
@@ -63,7 +71,12 @@ in
     boot.loader.grub.extraPerEntryConfig = mkIf (!cfg.hvm) "root (hd0)";
     boot.loader.grub.efiSupport = cfg.efi;
     boot.loader.grub.efiInstallAsRemovable = cfg.efi;
-    boot.loader.timeout = 0;
+    boot.loader.timeout = 1;
+    boot.loader.grub.extraConfig = ''
+      serial --unit=0 --speed=115200 --word=8 --parity=no --stop=1
```

Comment:
Lately I was wondering if we should make this the default for grub?
It's quite useful for servers with IPMI too.
cc @samueldr

Reply:
I would at least introduce a  `boot.loader.grub.serial.enable` option rather duplicating this everywhere.

Reply:
Not sure about enabling by default, as I don't recall whether GRUB will forcibly print a problematic warning/error message like it does for some "non-error" cases.

Though +1 for an option.

Reply:
I don't think this is universally correct, so I'd hesitate to make it a `serial.enable` option. I know of some bare metal which uses other versions of this.

Reply:
I'm going to merge as-is, we could make another PR around fancifying the serial config? Luckily this won't be so much of an annoyance soon as EFI obviates this stuff.